frontend/src/utils/string.ts

```diff
@@ -1,5 +1,13 @@
+import { RoleType } from '../../../shared/types/customTypes';
+
 export const titleCase = (s: string): string =>
   s.substr(0, 1).toLocaleUpperCase() + s.substr(1);
 
 export const percent = (maximumFractionDigits?: number) =>
   new Intl.NumberFormat(undefined, { style: 'percent', maximumFractionDigits });
+
+export const getRoleType = (role: string) => {
```

Comment:
This would become unnecessary if the role types aren't handled as strings (other than rendering purposes)